src/repl/scala/tools/nsc/interpreter/package.scala

```diff
@@ -199,10 +199,11 @@ package object interpreter extends ReplConfig with ReplStrings {
   }
 
   /* debug assist
+  debug assist */
   private[nsc] implicit class `smart stringifier`(val sc: StringContext) extends AnyVal {
     import StringContext._, runtime.ScalaRunTime.stringOf
     def ss(args: Any*): String = sc.standardInterpolator(treatEscapes, args map stringOf)
-  } debug assist */
+  }
```

Comment:
I'm not sure what's the meaning of these comments, but after uncommenting this class, maybe this empty comment should go away as well?